### PR TITLE
prompts edit + cerbathon category

### PR DESCRIPTION
**Pull request by unmoving-javi**
Messed with the JS for the Cerbathon a bit to add the Cerbathon prompts, fingers crossed that doesn't break anything. Maybe double check on that though.

Aside from that, added a few prompts and clarified one of them.

### CODE REVIEW

File: src/prompts/characters/cerber.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ export const regularPrompts = [
   'mods have to remind about redeems',
   'appreciating (not\xa0MILM)',
   '36% moment',
+  'says \'yippee\' unprompted',
```
Comment (cjmaxik) [inline src/prompts/characters/cerber.js]:
```suggestion
  'says "yippee" unprompted',
```

File: src/prompts/characters/cerber.js [hunk 2]
```diff
@@ -81,6 +85,20 @@ export const rareRedeemPrompts = [
...
+  'stream ends, timer has increased since start',
+  'cerbathon splash animation more than once',
+  'cerber fact of the day',
+  'cerber can\'t decide what to do',
+  '\"your timer is broken, it\'s only going up\"',
+  //'bingo metagaming' <- this one is kinda unclear & rare so idk
+  'lore video premiere',
+  'just chatting until 2 hour mark',
+  'staring contest'
+
```
Comment (cjmaxik) [inline src/prompts/characters/cerber.js]:
```suggestion
```

File: src/prompts/characters/cerber.js [hunk 3]
```diff
@@ -81,6 +85,20 @@ export const rareRedeemPrompts = [
...
+export const cerbathonPrompts = [
+  // cerbathon specific
+  'stream ends, timer has increased since start',
+  'cerbathon splash animation more than once',
+  'cerber fact of the day',
+  'cerber can\'t decide what to do',
+  '\"your timer is broken, it\'s only going up\"',
+  //'bingo metagaming' <- this one is kinda unclear & rare so idk
+  'lore video premiere',
+  'just chatting until 2 hour mark',
```
Comment (cjmaxik) [inline src/prompts/characters/cerber.js]:
```suggestion
  'just chatting until 2\xa0hour mark',
```

File: src/prompts/characters/cerber.js [hunk 4]
```diff
@@ -81,6 +85,20 @@ export const rareRedeemPrompts = [
   'stretch redeem'
 ]
 
+export const cerbathonPrompts = [
+  // cerbathon specific
+  'stream ends, timer has increased since start',
+  'cerbathon splash animation more than once',
+  'cerber fact of the day',
```
Comment (cjmaxik) [inline src/prompts/characters/cerber.js]:
```suggestion
  'cerber fact of the\xa0day',
```

**Comment by unmoving-javi** [comment]
Suggestions added! Looks good to me so far

**Pull request merged**